File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex22.java
import java.util.Scanner;

public class Ex22 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite um numero");
        var numero = scan.nextInt();

        if (numero % 2 == 0) {
            System.out.println("Par");
        } else {
            System.out.println("Impar");
        }

        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/c/Ex22.cs
using System;

class Ex22
{
	static void Main(string[] args)
	{
		Console.WriteLine("Digite um numero inteiro");
		int numero = int.Parse(Console.ReadLine());

		if (numero % 2 == 0)
		{
			Console.Write("Par");
		}
		else
		{
			Console.Write("Impar");
		}
	}
}
